backend/index.ts

```diff
@@ -0,0 +1,16 @@
+import cors from 'cors';
+import express from 'express'

```

Comment:
You should add missing semicolons in multiple pleaces. Include ESLint to your project in order to force semicolon usage.